src/js/user-profile/reducers/profile.js

```diff
@@ -15,14 +16,21 @@ const initialState = {
   gender: null,
   accountType: null,
   savedForms: [],
-  prefillsAvailable: []
+  prefillsAvailable: [],
+  loading: true
```

Comment:
It's also set to false if we don't have a user token. Any time the logged in status is updated, essentially.